samples/net/mqtt/src/modules/transport/Kconfig.transport

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig MQTT_SAMPLE_TRANSPORT
+	bool "Transport"
+	select MQTT_HELPER
+	select MQTT_CLEAN_SESSION
+	select HW_ID_LIBRARY
+	default y
```

Comment:
Not sure this should even be configurable. Without this module, the sample is more or less useless. Maube make it a promptless option and just use `menu "Transport"` instead

Reply:
My thinking is that the sample should be as modular as possible. Atleast the possibility of it. Lets say that a user wants to quickly disable the module in favour of using their own custom transport. I can see your points though. It gives the impression that the module is optional.